db/migrations/20190726141224_create_flap_table_and_triggers.sql

```diff
@@ -2,6 +2,9 @@
 CREATE TABLE maker.flap
 (
     id           SERIAL PRIMARY KEY,
+    -- TODO: remove hash?
+    -- Can we replace block number + hash with header_id?
+    -- Tricky because there might be multiple diffs contributing to a single row
```

Comment:
Yeah, going to leave this to the trigger-table-reorg-safety stories. I think it _probably_ makes sense to remove hash and number and then fetch those things based on `header_id` at query time, so that we're not duplicating the data - but re-org safety and query performance are probably higher priorities that merit deeper consideration before making the change